lib/AboutPreferences.jsm

```diff
@@ -269,77 +278,9 @@ this.AboutPreferences = class AboutPreferences {
         subcheck.classList.add("indent");
         subcheck.setAttribute("label", formatString(nested.titleString));
         linkPref(subcheck, nested.name, "bool");
-        if (nested.name === PREF_SHOW_SPONSORED) {
-          sponsoredStoriesCheckbox = subcheck;
-        }
       });
     });
 
-    if (discoveryStreamConfig.enabled) {
-      // If Discovery Stream is enabled hide Home Content options
-      contentsGroup.style.visibility = "collapse";
-
-      const discoveryGroup = homeGroup.insertAdjacentElement("afterend", homeGroup.cloneNode());
-      discoveryGroup.id = "discoveryContentsGroup";
-      discoveryGroup.setAttribute("data-subcategory", "discovery");
-      createAppend("label", discoveryGroup)
-        .appendChild(document.createElementNS(HTML_NS, "h2"))
-        .textContent = formatString("prefs_content_discovery_header");
-      const descriptionHbox = createAppend("hbox", discoveryGroup);
-      const discoveryGroupDescription = createAppend("description", descriptionHbox);
-      discoveryGroupDescription.textContent = formatString("prefs_content_discovery_description");
-      discoveryGroupDescription.classList.add("tail-with-learn-more");
-
-      // Add the Learn more link in the description
-      const topstoriesSection = prefStructure.find(s => s.id === "topstories");
-      const learnMoreURL = topstoriesSection && topstoriesSection.learnMore.link.href;
-      const link = createAppend("label", descriptionHbox);
-      link.classList.add("learn-sponsored");
-      link.classList.add("text-link");
-      link.setAttribute("href", learnMoreURL);
-      link.textContent = formatString("prefs_topstories_sponsored_learn_more");
-
-      if (discoveryStreamConfig.show_spocs && sponsoredStoriesCheckbox) {
-        sponsoredStoriesCheckbox.remove();
-        sponsoredStoriesCheckbox.classList.remove("indent");
-        discoveryGroup.appendChild(sponsoredStoriesCheckbox);
-      } else if (discoveryStreamConfig.show_spocs) {
-        // If there is no element to reuse create one
-        const discoveryDetails = createAppend("vbox", discoveryGroup);
-        const subcheck = createAppend("checkbox", discoveryDetails);
-        subcheck.setAttribute("label", formatString("prefs_topstories_options_sponsored_label"));
-        linkPref(subcheck, PREF_SHOW_SPONSORED, "bool");
-      }
-
-      const contentDiscoveryButton = document.createElementNS(HTML_NS, "button");
-      contentDiscoveryButton.classList.add("contentDiscoveryButton");
-      contentDiscoveryButton.textContent = formatString("prefs_content_discovery_button");
```

Comment:
Oops we forgot to remove this string:
https://github.com/mozilla/activity-stream/blob/da0171e0ed567c660f624a09ceb4ba6dfc71ccc6/locales-src/en-US/strings.properties#L97

Reply:
Oh and `formatString("prefs_content_discovery_header");` too ?

Reply:
ah, good catch